src/hooks/http.test.js

```diff
@@ -1270,9 +1270,6 @@ describe('http hook', () => {
     expect(Http.scrubQueryParams('?password=123&plain=123&secret=123')).toEqual(
       '?password=****&plain=123&secret=****'
     );
-    process.env[LUMIGO_SECRET_MASKING_REGEX_HTTP_QUERY_PARAMS] = '["plain"]';
```

Comment:
Is this test no longer required?

Reply:
It's redundant. That is already tested extensively in node-core, and it would take an overhaul of the tests using jest.isolateModules to test effectively because of the way node-core works.